crates/bevy_render/src/mesh/mesh/mod.rs

```diff
@@ -78,50 +74,82 @@ pub struct Mesh {
 ///         1, 3, 2
 ///     ])));
 ///     mesh
-///     // For further visualization, explanation, and examples see the built-in Bevy examples
-///     // and the implementation of the built-in shapes.
 /// }
 /// ```
-/// Common points of confusion:
-/// - UV maps in Bevy are "flipped", (0.0, 0.0) = Top-Left (not Bot-Left like `OpenGL`)
-/// - It is normal for multiple vertices to have the same position
-///   attribute - it's a common technique in 3D modelling for complex UV mapping or other calculations.
 ///
-/// To render correctly with `StandardMaterial` a mesh needs to have properly defined:
-/// - [`UVs`](Mesh::ATTRIBUTE_UV_0): Bevy needs to know how to map a texture onto the mesh.
-/// - [`Normals`](Mesh::ATTRIBUTE_NORMAL): Bevy needs to know how light interacts with your mesh. ([0.0, 0.0, 1.0] is very
-///              common for simple meshes because simple meshes are smooth, and they don't require complex light calculations.)
-/// - Vertex winding order -
-///              the default behavior is with `StandardMaterial.cull_mode` = Some([`Face::Front`](crate::render_resource::Face::Front)) which means
-///              that by default Bevy would *only* render the front of each triangle, and the front
-///              is the side of the triangle in which the vertices appear in a *counter-clockwise* order.
+/// You can see how it looks like [here](https://github.com/bevyengine/bevy/blob/main/assets/dovs/Mesh.png),
+/// used in a `PbrBundle` with a square bevy logo texture, with added axis, points,
+/// lines and text for clarity.
+///
+/// For further visualization, explanation, and examples see the built-in Bevy examples,
+/// in particular [generate_custom_mesh](https://github.com/bevyengine/bevy/blob/main/examples/3d/generate_custom_mesh.rs),
+/// and the [implementation of the built-in shapes](https://github.com/bevyengine/bevy/tree/main/crates/bevy_render/src/mesh/shape).
+///
+/// ## Common points of confusion
 ///
+/// - UV maps in Bevy are "flipped", `[0.0, 0.0]` = Top-Left (not Bot-Left like `OpenGL`).
```

Comment:
It's only flipped compared to OpenGL, to me this implies bevy is doing something non-standard here. As far as I know, top-left is part of the gltf spec

In other words, I agree it's worth pointing out, but it should be clearer that it's flipped compared to opengl.

Something like
```suggestion
/// - UV maps in bevy start at the top-left unlike opengl which starts at the bottom-left
```